source/SoundEmojiSynthesizer.cpp

```diff
@@ -207,10 +207,7 @@ ManagedBuffer SoundEmojiSynthesizer::pull()
                 done = true;
                 if (renderComplete || status & EMOJI_SYNTHESIZER_STATUS_STOPPING)
                 {
-                    // Wait for a hanful of milliseconds while we finish playing the last buffer we sent out ...
-                    //fiber_sleep( buffer.length() / sampleRate ); // Removed due to causing issues for Python
-
-                    // ... THEN flip out status and fire the event
+                    // Flip out status and fire the event
```

Comment:
I'm not sure I understand that comment?

Reply:
Fixed the typo, for clarity: https://github.com/lancaster-university/codal-microbit-v2/pull/229/commits/b5236e0ddc42dac33c9a3779dc6c3fd46812a505